packages/transport/test/modules/tokens/TokenController.test.ts

```diff
@@ -108,11 +108,12 @@ describe("TokenController", function () {
         expect((receivedToken.cache?.content as TokenContentFile).secretKey.toBase64()).toBe((sentToken.cache?.content as TokenContentFile).secretKey.toBase64());
     });
 
-    test("should send and receive a TokenContentRelationshipTemplate", async function () {
+    test("should send and receive a personalized TokenContentRelationshipTemplate", async function () {
```

Comment:
also here - ADD a test instead of replacing another one.

Reply:
I don't have a strong opinion in this case, but on a general basis: Would this approach make test files larger and thus make it more difficult to keep track of what is currently tested?

Reply:
you could screw up sending and receiving un-personalized TokenContentRelationshipTemplate and do not recognize it, because you deleted the test for it.

I have a really strong opinion on that.